.github/workflows/nodejs.yml

```diff
@@ -50,11 +50,6 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
 
```

Comment:
do we not need to use npm 6 any more? are the more recent version on npm AOK now?